### ISSUE TITLE
Assignment 01

### ISSUE THREAD

**Issue by jeehoonkang** [snu-sf-class/pl2016#22]
-  Assignment 01 is issued: https://github.com/snu-sf-class/pl2016#assignments
- Read the fine manual: https://github.com/snu-sf-class/pl2016#assignment
- Please ask questions here for the Assignment 01.

**Comment by swchoi06** [comment]
In P02.v in assignment 01, it says

> Use [Fixpoint] to define it. 

However.. blt_nat is defined with 'Definition', not 'Fixpoint'.

```

(** **** Problem #3: 2 stars (blt_nat) *)
(** The [blt_nat] function tests [nat]ural numbers for [l]ess-[t]han,
    yielding a [b]oolean.  Use [Fixpoint] to define it. *)

Definition blt_nat (n m : nat) : bool :=
  GIVEUP.

```

Is this your mistake?

**Comment by jeehoonkang** [comment]
Yeah.  Please change `Definition` into `Fixpoint` and define it.  The evaluation does not distinguish `Definition` and `Fixpoint`.

**Comment by jeehoonkang** [comment]
I will run the office hour every Thursday (https://github.com/snu-sf-class/pl2016/blob/master/README.md) after the class.
